test_wrappers.py

```
@@ -5381,10 +5385,13 @@ def test_ensembl_sequence_chromosome(run):
     )
 
 
-def test_ensembl_sequence_chromosomes(run):
-    run(
+def test_ensembl_sequence_multiple_chromosomes(run):
+    tmp_test_dir = run(
```

Ruff (0.8.2)

Local variable `tmp_test_dir` is assigned to but never used

Remove assignment to unused variable `tmp_test_dir`

(F841)